packages/ffmpeg/server/FFmpegExporterServer.ts

```diff
@@ -64,13 +64,15 @@ export class FFmpegExporterServer {
   private readonly promise: Promise<void>;
   private readonly settings: FFmpegExporterSettings;
   private readonly jobFolder: string;
+  private audioFilenames: string[];
```

Comment:
I don't think we ever set to audioFilenames outside the constructor so we should be able to make this readonly too.